spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -186,7 +185,7 @@ public void testCustomPartitionCountOverridesDefaultIfLarger() throws Exception
 		// Let the consumer actually bind to the producer before sending a msg
 		binderBindUnbindLatency();
 		moduleOutputChannel.send(message);
-		Message<?> inbound = moduleInputChannel.receive(2000);
+		Message<?> inbound = moduleInputChannel.receive((int)(2000 * timeoutMultiplier));
```

Comment:
While this improves the configurability of the test (which is good) - it adds something that will need to be copied in any newly added test (not that the hardcoded values were all that good to begin with). The first impulse is to perhaps isolate the constant and the multiplier in a utility method available to testers - e.g. receive(PollableChannel).